### PR TITLE
update ticdc faq

### PR DESCRIPTION
**Pull request by sdojjy**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-faq.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
 
 ## 在两个异地 TiDB 集群之间同步数据，如何部署 TiCDC？
 
-建议将 TiCDC 部署在下游 TiDB 集群。这是因为，如果上下游网络延迟较大，例如超过 100 ms 时，由于 MySQL 传输协议的原因，TiCDC 向下游执行 SQL 的延迟会急剧增加，导致系统的吞吐下降。部署在下游能够极大缓解该问题。
+建议将 TiCDC 部署在下游 TiDB 集群。这是因为，如果上下游网络延迟较大，例如超过 100 ms 时，由于 MySQL 传输协议的原因，TiCDC 向下游执行 SQL 的延迟会急剧增加，导致系统的吞吐下降。部署在下游能够极大缓解该问题。经过优化后，v6.5.2 及之后可版本以将 TiCDC 部署在上游。
```
Comment (nongfushanquan) [inline ticdc/ticdc-faq.md:261]: 经过优化后，v6.5.2 及之后的版本建议将 TiCDC 部署在上游集群。

Comment (sdojjy) [reply ticdc/ticdc-faq.md]: done

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -258,7 +258,7 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
 
 ## 在两个异地 TiDB 集群之间同步数据，如何部署 TiCDC？
 
-建议将 TiCDC 部署在下游 TiDB 集群。这是因为，如果上下游网络延迟较大，例如超过 100 ms 时，由于 MySQL 传输协议的原因，TiCDC 向下游执行 SQL 的延迟会急剧增加，导致系统的吞吐下降。部署在下游能够极大缓解该问题。
+建议将 TiCDC 部署在下游 TiDB 集群。这是因为，如果上下游网络延迟较大，例如超过 100 ms 时，由于 MySQL 传输协议的原因，TiCDC 向下游执行 SQL 的延迟会急剧增加，导致系统的吞吐下降。部署在下游能够极大缓解该问题。经过优化后，v6.5.2 及之后的版本建议将 TiCDC 部署在上游集群。
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:261]: 请修改为类似这样的，因为这是 v6.5 之后的文档，所以可以省略前面提到的 建议将 TiCDC 部署在下游 TiDB 集群。

```suggestion
从 v6.5.2 起，建议将 TiCDC 部署在上游集群。这是因为 xxxx

```

Comment (qiancai) [inline ticdc/ticdc-faq.md:261]:
```suggestion
对于 v6.5.2 之前的版本，建议将 TiCDC 部署在下游 TiDB 集群。这是因为，如果上下游网络延迟较大，例如超过 100 ms 时，由于 MySQL 传输协议的原因，TiCDC 向下游执行 SQL 的延迟会急剧增加，导致系统的吞吐下降。部署在下游能够极大缓解该问题。经过优化后，v6.5.2 及之后的版本建议将 TiCDC 部署在上游集群。
```

**Review by qiancai — APPROVED** [review]

**Review by nongfushanquan — APPROVED** [review]

**Pull request merged**